### PR TITLE
Minor bug fixes for issues.

### PR DESCRIPTION
**Pull request by wonderslug**
This commit has some minor bug fixes for issues and issue handling at the top level
validate command.
 
Signed-off-by: Brian Towles <btowles@cloudera.com>

### CODE REVIEW

File: cdpctl/validation/__init__.py [hunk 1]
```diff
@@ -121,7 +120,9 @@ def __init__(
     @property
     def message(self):
         """Get the message."""
-        return self._template.summary.format(*self._subjects)
+        if self._subjects:
```
Comment (aakanser) [inline cdpctl/validation/__init__.py:123]: do `self._template` and `self._template.summary` always exist?

**Review by aakanser — APPROVED** [review]
looks good, left one comment

File: cdpctl/validation/__init__.py (hunk 1 above)
Comment (wonderslug) [reply cdpctl/validation/__init__.py:123]: yes The template with them has to be passed as part of the Issue constructor.

Comment (wonderslug) [reply cdpctl/validation/__init__.py:123]: And summary is a required field for the IssueTemplate being pulled out of the issue_templates.yml

**Review by mahendra-rk — APPROVED** [review]

**Pull request merged**